backend/dataall/db/api/pipeline.py

```diff
@@ -237,6 +237,27 @@ def query_user_pipelines(session, username, groups, filter) -> Query:
                     models.DataPipeline.label.ilike(filter.get('term') + '%%'),
                 )
             )
+        if filter and filter.get('region'):
+            if len(filter.get('region')) > 0:
+                query = query.filter(
+                    or_(
```

Comment:
A very minor thing, this "or_" is not needed, you are only filtering on one condition

Reply:
Good call out! - resolved